Node-Task-7/controllers/users.controller.js

```diff
@@ -0,0 +1,44 @@
+const logger = require("../config/logger.config");
+const responce = require("../utils/response.utils");
+const userServices = require("../services/users.services");
+
+const register = async (req, res) => {
+  try {
+    const { username, password } = req.body;
+    if (!(username && password)) {
+      res.status(400).json({ message: "Username and Password is required" });
```

Comment:
Move responses construction to a separate util from where a response object can be constructed

Reply:
completed